### DISCUSSION TITLE
Unified schema for hospital prices and negotiated rates

### DISCUSSION THREAD

**Discussion by altosaar** [TPAFS/transparency-data#14] [General]
Love this effort! Thank you for the excellent README :)

We are building similar tools for working with this data at @onefact and would love to see what working with @TPFAS could look like. For example, we are running a data bounty with @dolthub to pay for some of this data to be collected: https://www.dolthub.com/repositories/onefact/paylesshealth/

We are also designing a unified schema & API for both hospital prices, negotiated rates (insurance prices), and associated health outcomes. 

Is a unified schema / API on your roadmap or are you aware of any schemas for this data that are public already? 

Thank you!!
Jaan

**Comment by MikeG112** [comment]
Thanks for the kind words Jaan, and for being here :)

Would love to talk more about how to facilitate collaboration between @onefact and @TPAFS. I've seen your data bounty and am super happy that you all tasked that crowdsourcing effort, it is great stuff!

I think unified schemas and standards that the community at large supports (with an eye towards applications that benefit patients) will be a huge boon to efforts like those of @TPAFS and @onefact, so I'm exited to hear you are working on that. One challenge that I think will be important to address is how to facilitate transforming whatever data is legally required and publicly reported into data that conforms to such community driven schema standards. Ideally this challenge would ultimately become minimal, and a solid, community driven schema would feedback and inform the @cmsgov choice for how to design reporting requirements in the future.

Schema iteration for each of the types of data we provide here, and developing standards and unified approaches for each, are definitely on the roadmap.

The most relevant schemas I am aware of are those developed by @cmsgov itself, and which dictate the reporting requirements for [health plan price transparency](https://www.cms.gov/healthplan-price-transparency). The schemas I'm referencing only relate to the insurer price transparency subset of the data housed in this repo, but for that particular case, you can see the relevant schemas [here](https://github.com/CMSgov/price-transparency-guide/tree/master/schemas). If you take a close look, you'll see that many of the schemas in the @TPAFS repo are informed by choices made in the the @cmsgov repo.

Is this the sort of thing you have in mind, or did I miss what you were really after? If this is the sort of thing you are seeking, this is the one instance I know of that is well-defined, and a standard (in at least the reporting sense); for example, the final rules for hospital price transparency have some data specific requirements, but they are far less well-defined, and there is no equivalent, mandated schema like there is in this health plan case.

Thanks for being here, and look forward to continuing the conversation!
